### PR TITLE
Show commit status in commit list row

### PR DESCRIPTION
**Pull request by rottencandy**
## Fixes 
https://issues.redhat.com/browse/HAC-2620

## Description
Show commit status in commit list rows in the commits list page.

## Type of change

- [ ] Feature
- [x] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## How to test or reproduce?
Go to commits list page.

## Browser conformance: 

- [ ] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/components/Commits/commit-status.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    let stat = '-';
+    plrsForCommit.some((plr) => {
+      const plrStatus = pipelineRunFilterReducer(plr);
+      if (statuses.includes(plrStatus)) {
+        stat = plrStatus;
+        return true;
+      }
+    }, {});
```
Comment (karthikjeeyar) [inline src/components/Commits/commit-status.ts:45]: This logic will return first matched pipelinerun's status against statuses array rather it should be the latest pipelinerun (with commit sha label) status.

eg: If i retrigger the pipelinerun, it still shows the old pipelinerun's status (succeeded)
Screenshot 2023-01-04 at 1 09 50 PM



```suggestion
    const plrStatus = pipelineRunFilterReducer(plrsForCommit[plrsForCommit.length - 1]);
    if (statuses.includes(plrStatus)) {
      stat = plrStatus;
      return stat;
    }
```

**Review by rohitkrai03 — APPROVED** [review]
/approve

**Pull request merged**